docs/docs/build/presence.md

```diff
@@ -110,6 +110,42 @@ Notifications API is partially implemented. All messages are always broadcast ev
 
 Notifications are fundamentally unreliable at this time as there are no built-in acknowledgements nor retained state. To prevent most common loss of notifications, always check for connection before sending.
 
-### Throttling
+### Throttling/grouping
```

Comment:
Nit (personal preference - feel free to ignore 😋):
```suggestion
### Throttling / Grouping
```

Reply:
Pre-existing content is 2:1 without spacing and 2:1 for caps:
  Create/connect separation
  Undo/Redo support
  Summary / Snapshot Tree Visualization
but I think I like the suggestion.